src/services/schemas/SchemasService.ts

```diff
@@ -68,7 +71,7 @@ export class SchemaService {
     let deferred: ng.IDeferred<void> = this.$q.defer<void>();
     this.findTarget(container.serverGroup)
       .then(coordinator => {
-        return this.dmrService.executePost({
+        this.dmrService.executePost({
```

Comment:
We need this return as it ensures that the result of the executePost is passed to the the chained `then.` call